index.js

```diff
@@ -16,11 +22,13 @@ Link.propTypes = {
 		PropTypes.arrayOf(PropTypes.node),
 		PropTypes.node
 	]).isRequired,
-	url: PropTypes.string
+	url: PropTypes.string,
+	fallback: PropTypes.bool
 };
 
 Link.defaultProps = {
-	url: ''
+	url: '',
+	fallback: true
```

Comment:
`showFallback`? Not sure if you have preference around boolean prop names.

Reply:
showFallback makes it sound like you’re actually triggering g showing the fallback. I think `fallback` is fine.